apps/client/src/pages/CheckerPage.js

```diff
@@ -61,6 +61,24 @@ const CheckerPage = ({ checker, matomoTrackEvent, resetChecker, topic }) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
+  const isActive = useCallback(
+    (component, finished = false) => {
+      // If component is only a string, we make it a array first
+      const allComponents = Array.isArray(component) ? component : [component];
+
+      // If finished is true we check if it's finished, else check activeComponents.
+      const components = finished ? finishedComponents : activeComponents;
+      return components.includes(...allComponents);
+    },
+    [activeComponents, finishedComponents]
+  );
+
+  // Redirect to Intro in case no session context has been found
+  if (!sessionContext[slug]) {
+    window.location.href = geturl(routes.intro, { slug });
+    return <LoadingPage />;
```

Comment:
Nice. We can make a new RedirectPage for this.

Reply:
I guess. What are you thinking of?
`return <RedirectPage href={} />` that basically does the same thing as these lines?
If so; can you create a separate task for this?

Reply:
Well `RedirectPage` already has other purposes (=OLO Redirect). So we need to make a new Page that receives a prop for the route (`href` or something indeed) and then use `window.location.href` to load this page and return a spinner (can be LoadingPage)